web/modules/custom/dpl_dashboard/src/Plugin/Block/DashboardBlock.php

```diff
@@ -79,22 +81,18 @@ public function build(): array {
 
     $data = [
       // Config.
-      'page-size-desktop' => $dashboardSettings->get('page_size_desktop'),
-      'page-size-mobile' => $dashboardSettings->get('page_size_mobile'),
-      'threshold-config' => $this->configFactory->get('dpl_library_agency.general_settings')->get('threshold_config'),
+      'page-size-desktop' => $dashboardSettings->get('page_size_desktop') ?? DplDashboardSettings::PAGE_SIZE_DESKTOP,
+      'page-size-mobile' => $dashboardSettings->get('page_size_mobile') ?? DplDashboardSettings::PAGE_SIZE_MOBILE,
+      'threshold-config' => $this->configFactory->get('dpl_library_agency.general_settings')->get('threshold_config') ?? GeneralSettingsForm::THRESHOLD_CONFIG,
       'interest-periods-config' => DplReactAppsController::getInterestPeriods(),
-      'reservation-detail-allow-remove-ready-reservations-config' => $generalSettings->get('reservation_detail_allow_remove_ready_reservations_config'),
+      'reservation-detail-allow-remove-ready-reservations-config' => $generalSettings->get('reservation_detail_allow_remove_ready_reservations') ?? GeneralSettingsForm::RESERVATION_DETAIL_ALLOW_REMOVE_READY_RESERVATIONS,
       'blacklisted-pickup-branches-config' => DplReactAppsController::buildBranchesListProp($this->branchSettings->getExcludedReservationBranches()),
       'branches-config' => DplReactAppsController::buildBranchesJsonProp($this->branchRepository->getBranches()),
 
       // Urls.
-      'dpl-cms-base-url' => DplReactAppsController::dplCmsBaseUrl(),
-      'fees-page-url' => '/user/me/fees',
+      // Cannot find that route. Does it exist?
       'intermediate-url' => '/user/me/intermediates',
-      'reservations-url' => '/user/me/reservations',
-      'physical-loans-url' => '/user/me/loans',
-      'search-url' => DplReactAppsController::searchResultUrl(),
-      'ereolen-my-page-url' => $generalSettings->get('ereolen_my_page_url'),
+      'ereolen-my-page-url' => dpl_react_apps_format_app_url($generalSettings->get('ereolen_my_page_url'), GeneralSettingsForm::EREOLEN_MY_PAGE_URL),
```

Comment:
Please remember to add a dependency on `dpl_react_apps` and `dpl_library_agency` when we use classes and functions from these modules.

Reply:
👍